src/Sulu/Bundle/ContentBundle/Repository/ResourceLocatorRepository.php

```diff
@@ -69,15 +55,10 @@ public function generate($parts, $parentUuid, $uuid, $webspaceKey, $languageCode
         $structure = $this->structureManager->getStructure($templateKey);
         $title = $this->implodeRlpParts($structure, $parts);
 
-        if ($parentUuid !== null) {
-            $parentDocument = $this->documentManager->find($parentUuid, $languageCode, ['load_ghost_content' => false]);
-            $parentPath = $this->rlpStrategy->loadByContent($this->getPublishedAncestorOrSelf($parentDocument));
-            $result = $this->rlpStrategy->generate($title, $parentPath, $webspaceKey, $languageCode, $segmentKey);
-        } elseif ($uuid !== null) {
+        if ($uuid !== null) {
```

Comment:
Why do we still need the `$parentUuid` and the `$uuid`? The page we are generating the URL for never has an own UUID, so I don't really get why we need two UUIDs...